lore/settings.py

```diff
@@ -289,6 +291,13 @@ def get_var(name, default):
     },
 }
 
+REST_FRAMEWORK = {
```

Comment:
I have no problem with this, but does it need to get any of this from an environment variable, @carsongee?